Dockerfile

```diff
@@ -16,8 +16,8 @@ RUN cd /usr/local/gtclang_build
     make -j2
 
 FROM nvidia/cuda:10.0-devel-ubuntu18.04
-MAINTAINER Carlos Osuna <Carlos.Osuna@meteoswiss.ch>
-LABEL description="This is a gtclang container"
+LABEL maintainer="Carlos Osuna <Carlos.Osuna@meteoswiss.ch>"
```

Comment:
ups, I thought I already did that following your comment in the previous PR. Something went wrong.